velox/connectors/hive/storage_adapters/hdfs/HdfsFileSystem.h

```diff
@@ -71,6 +80,10 @@ class HdfsFileSystem : public FileSystem {
   }
 
   static bool isHdfsFile(std::string_view filename);
+  static HdfsServiceEndpoint getServiceEndpoint(const Config* config);
```

Comment:
We don't need this as a separate function. The factory does not use this.
Add a comment describing that filePath is used to infer the endpoint. If an identity is missing from the filePath,  then the config is used.

Reply:
Thanks for your comment!
We have some unit tests to test the config path, with this separate function used. We can just keep it, right?

Reply:
Just added some description comments. Thanks!

Reply:
Does not help to add this API for testing. We should test the new `getServiceEndpoint` API with a path that has no identity and ensure it works as expected.

Reply:
Yes, I just removed this useless API in the new commit. Thanks!